domain.yml

```diff
@@ -1,36 +1,40 @@
 intents:
-- greet
-- goodbye
-- mood_affirm
-- mood_deny
-- mood_great
-- mood_unhappy
+  - greet
```

Comment:
Don't you usually explicitly remove this indent in yaml lists?

Reply:
haahaa no 😄 I've got this habit of running Option+CMD+L in pycharm for auto-formatting, so that just what happens to YAML files